src/dhke.rs

```diff
@@ -34,9 +35,9 @@ where
         self.0.as_bytes()
     }
 }
-
+#[cfg(feature = "zero")]
 impl<P> Zeroize for DiffieHellmanSharedSecret<P>
-where P: Zeroize
+where P: PublicKey
```

Comment:
I'm probably missing something here, but how does this even compile if there's no `Zeroize` trait bound on `P` at any time?